src/client/Microsoft.Identity.Client/Platforms/Android/AndroidBroker.cs

```diff
@@ -213,5 +213,27 @@ private static MsalTokenResponse CreateMsalTokenResponseFromResult(string broker
 
             return MsalTokenResponse.CreateFromBrokerResponse(response);
         }
+
+        public async Task<IEnumerable<IAccount>> GetAccountsAsync(string clientID)
+        {
+            Dictionary<string, string> brokerPayload = new Dictionary<string, string>();
+            brokerPayload.Add(BrokerParameter.ClientId, clientID);
+
+            try
+            {
+                await _brokerHelper.InitiateBrokerHandshakeAsync(_activity).ConfigureAwait(false);
+
+                return _brokerHelper.GetBrokerAccountsInAccountManager(brokerPayload);
+            }
+            catch (Exception ex)
+            {
+                _logger.Error("Broker Operation Failed to complete. In order to perform brokered authentication on android" +
```

Comment:
>_logger [](start = 16, length = 7)

shouldn't this be something more like "silent auth w/android broker failed because there were no accounts in the account manager"?

Reply:
no, this code is only designed to get the accounts from the broker. no authentication here.

Reply:
However, i can update the message to be more specific to get accounts